### PR TITLE
mac m1 chip installation instructions

### PR DESCRIPTION
**Pull request by marharyta**
- [x ] Update docs/ (if applicable)

As I was working on the project with Mac M1, this is what I discovered during the installation

### CODE REVIEW

File: docs/development.md [hunk 1]
```diff
@@ -128,6 +128,13 @@ To conveniently run all the tests you can run the lein alias
 ```
 lein alltests
 ```
+### Automated accessibility test report
```
Comment (opqdonut) [inline docs/development.md:131]: did you mean to put this here? now "Automated accessibility test report" is under "Development database" which sounds a bit weird

File: docs/development.md [hunk 2]
```diff
@@ -145,10 +152,11 @@ components in various configurations and is useful for example to develop and do
 
 See a running guide as example [https://rems-dev.rahtiapp.fi/guide](https://rems-dev.rahtiapp.fi/guide).
 
-## Automated accessibility test report
 
-We use [axe](https://www.deque.com/axe/) for automated accessibility tests.
+## Mac OS installation for Apple M1 Chip (Apple Silicon M1)
```
Comment (opqdonut) [inline docs/development.md:156]: How about having something like `## Depedencies` at the top of this file that would mention you need leiningen and java <15, with a subsection for M1?

File: docs/development.md [hunk 3]
```diff
@@ -145,10 +152,11 @@ components in various configurations and is useful for example to develop and do
...
+
+    1. Java Installation: install Java via Azul, make sure you download the version of [Java for MacOS for ARM architecture and not newer then version 13](https://www.azul.com/downloads/zulu-community/?version=java-13-mts&os=macos&architecture=arm-64-bit&package=jdk). 
+ 
+    2. Do not install leiningen through homebrew, rather use the script from [here](https://purelyfunctional.tv/guide/how-to-install-clojure/#mac-leiningen) to install leiningen. Run `lein update 2.8.3` and `lein version` and make sure the version is  2.8.3.
```
Comment (opqdonut) [inline docs/development.md:160]: lein 2.9.1 and 2.9.5 also work for me. I don't think we depend on any specific lein version. Did you have problems with fresh leiningen?

**Review by opqdonut — CHANGES_REQUESTED** [review]

**Comment by marharyta** [comment]
Will be rewriting this docs as per our last conversation

File: docs/development.md (hunk 1 above)
Comment (marharyta) [reply docs/development.md:139]: Isnt is under Running tests now?

File: docs/development.md (hunk 2 above)
Comment (marharyta) [reply docs/development.md:164]: sure, that works too

File: docs/development.md (hunk 1 above)
Comment (opqdonut) [reply docs/development.md:139]: you're right! I'm blind.

File: docs/development.md [hunk 4]
```diff
@@ -145,10 +160,11 @@ components in various configurations and is useful for example to develop and do
 
 See a running guide as example [https://rems-dev.rahtiapp.fi/guide](https://rems-dev.rahtiapp.fi/guide).
 
-## Automated accessibility test report
 
-We use [axe](https://www.deque.com/axe/) for automated accessibility tests.
+## Mac OS installation for Apple M1 Chip (Apple Silicon M1)
+
+    1. Java Installation: install Java via Azul, make sure you download the version of [Java for MacOS for ARM architecture](https://www.azul.com/downloads/zulu-community/?os=macos&architecture=arm-64-bit&package=jdk).
```
Comment (opqdonut) [inline docs/development.md:166]: I think you have too much indentation here: github is rendering this as a code block now.

File: docs/development.md [hunk 5]
```diff
@@ -1,5 +1,18 @@
 # Getting started with REMS development
 
+## Installed Software
+
+In order to get started with REMS, you need to have the following software installed:
+
+   - Docker
+   - Java. Currently REMS should be working with Java versions 11 to 16
+   - leiningen. As of 14/04/2021 the project should work with lein version Leiningen 2.9.5 on Java 16 OpenJDK 64-Bit Server VM.
```
Comment (opqdonut) [inline docs/development.md:9]: do we need to mention the java version here again? also, please consider a timestamp format 2021-04-14, which we use elsewhere (e.g. releases)

File: docs/development.md [hunk 6]
```diff
@@ -1,5 +1,18 @@
...
+### Mac OS installation for Apple M1 Chip (Apple Silicon M1)
+
+    1. Java Installation: install Java via Azul, make sure you download the version of [Java for MacOS for ARM architecture](https://www.azul.com/downloads/zulu-community/?os=macos&architecture=arm-64-bit&package=jdk).
```
Comment (opqdonut) [inline docs/development.md:12]: there's too much indentation here and github is rendering this as a code block you can just remove all indentation and it'll be fine

**Review by opqdonut — APPROVED** [review]
two suggestions, feel free to merge tho

**Comment by marharyta** [comment]
if all good, we could merge?

**Review by opqdonut — APPROVED** [review]

**Pull request merged**